client/src/scripts.js

```diff
@@ -0,0 +1,26 @@
+import { server_url } from './config.json'
```

Comment:
I'm not sure `scripts.js` is a good name for this module, something like `utils.js` sounds better.